setup.py

```diff
@@ -472,6 +472,7 @@ def maybe_cythonize(extensions, *args, **kwargs):
     """
     Render tempita templates before calling cythonize
     """
+    kwargs.setdefault('language_level', 2)
```

Comment:
There is a dict in this file which contains directives sent to cythonize. Think it would be more consistent if you set this therein (see PR you linked for sample of how)

Reply:
sure, moved and force-pushed.